plugin/core/sessions.py

```diff
@@ -1343,12 +1345,12 @@ def update_folders(self, folders: List[WorkspaceFolder]) -> None:
         if self.should_notify_did_change_workspace_folders():
             added, removed = diff(self._workspace_folders, folders)
             if added or removed:
-                params = {
+                params = cast(DidChangeWorkspaceFoldersParams, {
                     "event": {
                         "added": [a.to_lsp() for a in added],
                         "removed": [r.to_lsp() for r in removed]
                     }
-                }
+                })
```

Comment:
Preferred to have `# type` comment but the issue with `WorkspaceFolder` would have to solved.